typing_examples.py

```diff
@@ -0,0 +1,35 @@
+"""Some examples of prometheus-async typing integration."""
+from asyncio import Future
+
+from prometheus_client.metrics import Summary  # type: ignore
+
+from prometheus_async.aio import time
+
+
+REQ_DURATION = Summary("REQ_DUR", "Request duration")
+
+
+@time(REQ_DURATION)
+async def func(i: int) -> str:
+    return str(i)
+
+
+# The type of `func` is correct:
+# "def (i: builtins.int) -> typing.Coroutine*[Any, Any, builtins.str]"
+# reveal_type(func)
+
```

Comment:
Can't you do the `# type: ignore` trick here?

Reply:
Done, and added the Mypy option.